### PR TITLE
Paddle installation and evaluation

### PR DESCRIPTION
**Pull request by cheungdaven**

### CODE REVIEW

File: config.ini [hunk 1]
```diff
@@ -183,29 +186,34 @@ google_analytics_tracking_id = UA-96378503-2
...
               tensorflow, d2l-ai/d2l-zh-tensorflow-colab
+              paddle, d2l-ai/d2l-zh-paddle-colab
 
 replace_svg_url = img, http://d2l.ai/_images
 
 libs = mxnet, mxnet, -U mxnet-cu101==1.7.0
        mxnet, d2l, git+https://github.com/d2l-ai/d2l-zh@release  # installing d2l
        pytorch, d2l, git+https://github.com/d2l-ai/d2l-zh@release  # installing d2l
        tensorflow, d2l, git+https://github.com/d2l-ai/d2l-zh@release  # installing d2l
+       paddle, d2l, git+https://github.com/d2l-ai/d2l-zh@release
```
Comment (astonzhang) [inline config.ini:197]: Just append the same comment

**Comment by astonzhang** [comment]
Just created the following:
https://github.com/d2l-ai/d2l-zh-paddle-colab
https://github.com/d2l-ai/d2l-zh-paddle-sagemaker

@AnirudhDagar Feel free to merge it if this PR looks good to you.

**Comment by tngt** [comment]
> Just created the following: https://github.com/d2l-ai/d2l-zh-paddle-colab https://github.com/d2l-ai/d2l-zh-paddle-sagemaker
> 
> @AnirudhDagar Feel free to merge it if this PR looks good to you.
I would like to ask if this PR can be merged? If cannot merge temporarily, will it affect the submission of the d2l(paddle version). Looking forward to your reply。

**Comment by AnirudhDagar** [comment]
Hi @tngt sorry for the delay, I was waiting for @cheungdaven to address the requests by @astonzhang.
But since he is OOTO, I've made the change and will get this in as soon as CI is green again. Sorry to keep you blocked.

You can send in your changes then.

**Review by AnirudhDagar — APPROVED** [review]

**Pull request merged**